README.md

```diff
@@ -250,10 +250,9 @@ On master/main branch...
 
 1. Fork it
 2. Create your feature branch (`git checkout -b my-new-feature`)
-3. If you changed the javascript/scss files, run `yarn build`.
```

Comment:
I removed this because I believe it is no longer needed, correct me if I'm wrong @gmq